### ISSUE TITLE
Focus is unmanaged after activating the "Get Started" router link

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#3167]
## Description  
Focus is unmanaged after activating the "Get Started" router link. When using a desktop browser, focus is set to the <body> element after after activating the "Get Started" router link. When using VoiceOver/Safari/iOS 12.1.4, VoiceOver begins reading at the second paragraph of the newly loaded content rather then the beginning of the newly loaded content as expected.

## Location     
Home to Clarity Design page 

## Priority     
Medium

## Recommended fix      
Set focus to the heading at the beginning of the newly added content after the "Get Started" router link is activated.

## LOE  
Medium

13 | Activating the Get started   button from the home page puts the focus in the middle of the Clarity design   system page | Home to Clarity Design page |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | --

**Comment by AlexMarcuBytex** [comment]
Hi, is this issue solved? If not, @Shijir I see you integrated the focus directive in #3725 PR. Should be used here?

**Comment by Shijir** [comment]
Hi @AlexMarcuBytex! You are right. #3725 resolved this issue. I didn't know we have a separate issue here as well. Thanks for bringing this to my attention.
